packages/light-client-experimental/scripts/testing.ts

```diff
@@ -0,0 +1,116 @@
+import {
```

Comment:
If we want to commit this file as is, we should also make it clear how to actually run it (eg in the README or comment at the top of this file) ie the `pnpm dlx tsx ./scripts/testing.ts` step